airflow/configuration.py

```diff
@@ -309,15 +309,29 @@ def run_command(command):
 # listen (in seconds).
 job_heartbeat_sec = 5
 
-# The scheduler constantly tries to trigger new tasks (look at the
-# scheduler section in the docs for more information). This defines
-# how often the scheduler should run (in seconds).
+# When the scheduler is running, it periodically updates a timestamp in the DB
+# to indicate that it's running. This controls how often this timestamp is
+# updated.
 scheduler_heartbeat_sec = 5
 
+# The number of seconds the scheduler should run before exiting. It is expected
+# that the scheduler process will be restarted in a loop.
```

Comment:
(num_runs doesnt make sense, still the conf option should be named clearer I think)